marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -5,11 +5,11 @@ const AdvancedSettings = {
     'targetFormat': 'The format of the documents in the target database.',
     'additionalCollections': 'The collection tags to add to the default tags assigned to the processed document.',
     'targetPermissions': 'The comma-separated permissions required to access the processed document.',
-    'module': 'The path to your custom hook module.',
-    'cHParameters': 'Parameters, as key-value pairs, to pass to your custom hook module.',
-    'user': 'The user account to use to run the module. The default is the user running the flow; e.g., data-hub-operator.',
-    'runBefore': 'Choose whether to run the module before or after the core processes of the step.',
-    /* The following are for *Custom*Tooltips. */
+    'headers': 'A JSON object that represents additional metadata to add to the header section of the envelope of each document.',
+    'provGranularity': 'The level of detail logged for provenance. Choose *coarse* for the default level or *off* for no provenance logging.',
+    'processors': 'Custom modules that perform additional processes after the core step processes are completed and before the results are saved.',
```

Comment:
I didn't see a reference for these; if that's true, I'd like to use the following descriptions instead:

- Processors = Custom modules that can process content before it is persisted. 
- Custom hook = Custom module that can perform updates in a separate transaction. 

Describing how a custom hook behaves is very difficult since its behavior varies based on the step being used. So I think the less said here, the better. I really don't think that Pari - the typical user of a custom hook - is ever going to use this interface to configure processors/custom hook. The complexity in trying to describe what these are is one of the reasons I don't think these should be exposed here either. But since we are exposing them, I think we want brief descriptions that encourage a non-Pari persona to not bother trying to figure out what these are.